CHANGELOG.md

```diff
@@ -106,6 +106,10 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
   This method is used to resume a previously stopped services.
   (#1693)
 
+- `BlockProof` structure has been changed. The field `precommits` has a type
```

Comment:
Given BlockProof was added after the latest release, we could probably leave this change out.